### DISCUSSION TITLE
What is this object?

### DISCUSSION THREAD

**Discussion by 64Axes** [atom/atom#22147] [Q&A]
There is a strange box on my screen, it is annoying. Does anyone know what it is or how I can get rid of it?

![Screenshot_20210406_135806](https://user-images.githubusercontent.com/82059458/113772220-a5e52d00-96e1-11eb-83b8-78985a0f730a.png)

**Comment by sadick254** [comment]
Which OS are you running Atom on?

**Reply by ThatXliner** [reply]
This happens sometimes on macOS, too.

**Reply by 64Axes** [reply]
I am using Manjaro Linux.

**Comment by UziTech** [comment]
Are you able to inspect it with devtools?

1. <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>i</kbd>
2.  Click on the arrow in the top left.
![image](https://user-images.githubusercontent.com/97994/114075693-706c4b00-986b-11eb-981a-f1f15bfad379.png)
3. Click on the object.

**Reply by 64Axes** [reply]
`<div class="scrollbar-corner" style="position: absolute; height: 20px; width: 20px; bottom: 0px; right: 0px; overflow: scroll;"></div>
`
This is what it gave me.

**Reply by ThatXliner** [reply]
If it's an element, can't one just `display:none` it?

**Reply by UziTech** [reply]
It seems like it might be just an issue with styling. Maybe a package is overriding a themes styling for the `.scrollbar-corner`?
